DIRECTORY.md

```diff
@@ -641,6 +641,7 @@
 ## Web Programming
   * [Crawl Google Results](https://github.com/TheAlgorithms/Python/blob/master/web_programming/crawl_google_results.py)
   * [Current Stock Price](https://github.com/TheAlgorithms/Python/blob/master/web_programming/current_stock_price.py)
+  * [Current Weather](https://github.com/TheAlgorithms/Python/blob/master/web_programming/current_weather.py)
```

Comment:
It's Perfect

Reply:
That change was made automatically by
https://github.com/TheAlgorithms/Python/blob/master/.github/workflows/directory_writer.yml#L15 -->
https://github.com/TheAlgorithms/Python/blob/master/scripts/build_directory_md.py